app/src/main/java/lt/vilnius/tvarkau/entity/Problem.java

```diff
@@ -107,6 +110,8 @@ public String getId() {
         return id;
     }
 
+    public String getIdForVilniusAccount() { return idForVilniusAccount; }
```

Comment:
It's quite tempting  to use such boilerplate killer. Although googled a bit and found Jake Wharton comment that "I don't use it because it's implemented as a giant hack." ([link](https://www.reddit.com/r/androiddev/comments/2nneo7/anyone_using_project_lombok/)). Now I am confused. :)